compiler/src/dotty/tools/dotc/core/quoted/PickledQuotes.scala

```diff
@@ -75,7 +75,9 @@ object PickledQuotes {
     unpickled match {
       case Block(aliases, tpt) =>
         // `@quoteTypeTag type` aliasses are not required after unpickling
-        tpt
+        // `@quoteTypeTag type` aliasses are not required after unpickling
```

Comment:
Duplicate comment (or is GitHub's diff wrong here?)
Also: typo: "aliasses"

Reply:
Also: can you outline the reason why `quoteTypeTag` type aliases are not required after unpickling?

Reply:
I extended the comment with the reason it is not required